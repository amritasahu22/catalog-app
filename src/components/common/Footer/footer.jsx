import React from "react";

const Footer = () => {
	return (
		<footer className="text-muted">
			<div className="container">
				<p className="float-right">
					<a href="#">Back to top</a>
				</p>
				<p>Sample Women Clothing Catalogue &copy; 2021</p>
			</div>
		</footer>
	);
};

export default Footer;
