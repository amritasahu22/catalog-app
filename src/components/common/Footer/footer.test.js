import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Footer from "./footer";

test("should render properly", () => {
	const tree = shallow(<Footer />);
	expect(toJson(tree)).toMatchSnapshot();
});
