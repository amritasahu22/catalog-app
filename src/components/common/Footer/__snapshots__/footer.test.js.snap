// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render properly 1`] = `
<footer
  className="text-muted"
>
  <div
    className="container"
  >
    <p
      className="float-right"
    >
      <a
        href="#"
      >
        Back to top
      </a>
    </p>
    <p>
      Sample Women Clothing Catalogue Â© 2021
    </p>
  </div>
</footer>
`;
